src/core/public/overlays/modal/modal_service.tsx

```diff
@@ -128,6 +149,65 @@ export class ModalService {
 
         return modal;
       },
+      openConfirm: (message: MountPoint | string, options?: OverlayModalConfirmOptions) => {
+        // If there is an active modal, close it before opening a new one.
+        if (this.activeModal) {
+          this.activeModal.close();
+          this.cleanupDom();
+        }
```

Comment:
I'm using the same element for modals and confirmation dialogs, meaning that only one can be opened at a time. This was already the case for modals. If we want to be able to have a modal and a confirmation dialog opened at the same time, the service needs some more heavy changes, as they are both rendering inside a `EuiOverlayMask`, and we can't really render two of theses (would cause UI overlays/colors glitches).